stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -15,7 +15,10 @@ set(SWIFT_RUNTIME_CONCURRENCY_SWIFT_FLAGS -I${CMAKE_CURRENT_SOURCE_DIR}/Internal
 
 set(swift_concurrency_private_link_libraries)
 if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
-  list(APPEND swift_concurrency_private_link_libraries Synchronization)
+  list(APPEND swift_concurrency_private_link_libraries
+    Synchronization
+    mincore.lib # For QueryInterruptTime()
```

Comment:
Not kernel32.lib (per docs)?

Reply:
No. The docs for `QueryInterruptTime()` actually say it's in `mincore.lib`, but the ones for `QueryUnbiasedInterruptTime()` say `kernel32.lib`, however the latter is wrong (I dumped the symbols from `mincore.lib`; sure enough, it has both of them).

Reply:
We should probably report the documentation bug to Microsoft. :) The documentation is open-sourced [here](https://github.com/MicrosoftDocs/sdk-api/blob/b23c70c9092f31a0ff17bb0f60c03fff7855510c/sdk-api-src/content/realtimeapiset/nf-realtimeapiset-queryunbiasedinterrupttimeprecise.md).

Reply:
https://github.com/MicrosoftDocs/sdk-api/pull/1890

Reply:
Thanks!